src/services/tfjs-mobilenet-service.ts

```diff
@@ -73,8 +73,9 @@ interface MobilenetEmbedding extends MobilenetParams {
  * @return a reference number to the model, maintained by the `ResourceManager`.
  */
 const load = async ({version = 1, alpha = 1.0}: MobilenetParams): Promise<Reference> => {
-  log('Loading tfjs...');
-  const tf = await requireTf();
+  log('Loading tensorflow');
+  await requireTf();
```

Comment:
What happened to `const tf`?

Reply:
I removed, then added back, this require logic. I need not create a diff here, I'll revert the changes. 

As an aside: we don't really need to access any function or property within the tf module here. We merely want to dynamically load the module. That's why `const tf` went away.